### ISSUE TITLE
review use of heading tags throughout site

### ISSUE THREAD

**Issue by phette23** [code4lib/2020.code4lib.org#21]
E.g. workshops pages were using an `<h4>` for stylistic purposes, there seems to be inconsistency about what main heading is used for page titles.

**Comment by jscaffrey** [comment]
Live pages with skipped headings

Top level pages
1. [Home](https://2020.code4lib.org/) 2 skipped headings. Skips from H2 to H4. Preview looks fine without changing CSS.
2. [Schedule Timeline](https://2020.code4lib.org/schedule/timeline) Skips from H1 to H4. Preview looks OK without changing CSS. Alternately, PT Sans in `text-transform: uppercase;` and `font-weight: bold;` and a relatively smaller `font-size` could look good.
3. [Speakers](https://2020.code4lib.org/speakers/past-keynotes) Skips from H1 to H4. Preview looks fine without changing CSS.

Subpages
1. [Propose a Talk](https://2020.code4lib.org/general-info/propose-a-talk.html) Skips H1
2. [Propose a Workshop](https://2020.code4lib.org/general-info/propose-a-workshop.html) Skips H1

Related issue
- H2 and H3 not visually distinct, e.g. [](https://2020.code4lib.org/conduct/) and [](https://2020.code4lib.org/general-info/first-timers)

Passes Review for Skipped headers:
- [](https://2020.code4lib.org/general-info/scholarships)
- [](https://2020.code4lib.org/general-info/first-timers)
- [](https://2020.code4lib.org/general-info/about-c4l)
- [](https://2020.code4lib.org/prospectus/)

Not yet reviewed (Recommend review using dev data)
- https://2020.code4lib.org/schedule/day-1/ 
- https://2020.code4lib.org/schedule/day-2/
- https://2020.code4lib.org/schedule/day-3/

**Comment by dzoladz** [comment]
@jscaffrey 

> H2 and H3 not visually distinct, e.g. https://2020.code4lib.org/conduct/ and https://2020.code4lib.org/general-info/first-timers

I'm seeing that there is some conditional logic that could stand to be rethought. For example: on the `/first-timers` page, the `<h2>` element will be hidden if the social activities URL is not set.

```html
      {% if site.data.conf.social-activities-url != '' %}
      <h2>Social Activities</h2>
      <p>
        There's a lot going on outside of the sessions, and <a href="{{site.data.conf.social-activities-url}}">this page has information about various gatherings, events, meals, etc.</a>
        In addition, if you have any events that you’d like to organize, please feel free to add the event to the DIY Event Google Doc!
      </p>
       {% endif %}
```

We probably want `<h2>Social Activities</h2>` to display if any of the events is set to `show: true` in `conf.yaml`.

I can take a look at fixing it.

**Comment by jscaffrey** [comment]
Checked additional pages:

Missing H1:

- https://2020.code4lib.org/schedule/

Pages with skipped headings (in addition to the above)

- Skips from H1 to H4 https://2020.code4lib.org/talks/ 
- Skips from H1 to H4 https://2020.code4lib.org/workshops/
- Skips from H1 to H4 https://2020.code4lib.org/speakers/
- Skips from H2 to H4 https://2020.code4lib.org/schedule/day-1/
- Skips from H2 to H4 https://2020.code4lib.org/schedule/day-2/
- Skips from H2 to H4 https://2020.code4lib.org/schedule/day-3/

**Comment by kaitlinnewson** [comment]
Adding this here as noted in #22:

On https://2020.code4lib.org/schedule/, the times (e.g. "9AM - 5PM") are in an `<h4>` and the section names (e.g. "Preconference Workshops") are in an `<h3>` - I think these should be reversed (both based on DOM order and logical sections) and styled with CSS as appropriate.

**Comment by phette23** [comment]
For the pages that @jscaffrey lists, I think these templates are a matter of moving from `<h4>...</h4>` to `<h2 class="h4">...</h2>" to retain the same appearance but change level. PR incoming.

**Comment by jscaffrey** [comment]
In the process of closing this issue and breaking out into smaller issues.
